install_dependencies.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+sudo apt update && sudo apt install -y python3-dev python3-pip libglib2.0-dev
+
+# Install Bazel
+sudo apt install apt-transport-https curl gnupg -y
```

Comment:
Can you follow the best practices for apt-get install in docker files? https://docs.docker.com/develop/develop-images/instructions/#run

Reply:
Done, thanks for the comment.